seed.js

```diff
@@ -7,21 +7,21 @@ const Book = require('./models/book.js')
 async function seed(){
   const anthonyBook = new Book({
     title: 'Jurrasic Park',
-    desciption: 'A cautionary tale about genetic engineering, it presents the collapse of an amusement park showcasing genetically re-created dinosaurs to illustrate the mathematical concept of chaos theory and its real-world implications.',
+    description: 'A cautionary tale about genetic engineering, it presents the collapse of an amusement park showcasing genetically re-created dinosaurs to illustrate the mathematical concept of chaos theory and its real-world implications.',
     status: true,
     email: 'anthonymorton760@gmail.com'
 
   });
   const brianBook = new Book({
     title: 'Redwall',
-    desciption: 'Originally published in 1986, it is the first book of the Redwall series.',
+    description: 'Originally published in 1986, it is the first book of the Redwall series.',
     status: true,
     email: 'bkasprzyk@outlook.com'
 
   });
   const harveyBook = new Book({
     title: 'Odyssey',
-    desciption: 'The Odyssey is one of two major ancient Greek epic poems attributed to Homer',
+    description: 'The Odyssey is one of two major ancient Greek epic poems attributed to Homer',
     status: true,
     email: 'tripppdx@gmail.com'
```

Comment:
I see this is an async function. You may find that you you should use `await` here. 

`const seed = async () => {
await Book.create({})`